client/signup/controller.js

```diff
@@ -144,13 +146,6 @@ export default {
 			}
 		}
 
-		// Record a `calypso_*` tracks event after clicking the launch button.
-		// We record the event here because the launch button itself is managed by the wpcom-block-editor app
-		// wpcom-block-editor app may be treated differently by the ad blocker but this code is guaranteed to be hit.
-		if ( context.pathname === '/start/new-launch' ) {
-			recordTracksEvent( 'calypso_newsite_launch_start' );
-		}
-
```

Comment:
@johnHackworth did you mean to revert adding this event (https://github.com/Automattic/wp-calypso/pull/42188) in this PR or was this accident?